service/src/main/java/bio/terra/profile/service/crl/AzureCrlService.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.profile.service.crl;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.profile.app.configuration.AzureConfiguration;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.resourcemanager.costmanagement.CostManagementManager;
+import com.azure.resourcemanager.managedapplications.ApplicationManager;
+import com.azure.resourcemanager.resources.ResourceManager;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AzureCrlService {
```

Comment:
Does "CRL" stand for "Cloud Resources Library"?

I think I finally worked this out, but it would be helpful to just add some documentation for other folks who are acronym-challenged.